util/include/openssl_digest_creator.ipp

```diff
@@ -12,19 +12,18 @@
 #pragma once
```

Comment:
Why is this file named *.ipp?

Reply:
Resolved in https://github.com/arc-vmware/concord-bft/pull/5/commits/0e56648d6e0c220038d05b247a4bc589806a90a9